charts/oes/values.yaml

```diff
@@ -150,7 +155,7 @@ db:
   image:
     registry: quay.io/opsmxpublic
     repository: ubi8-oes-autopilot-db
-    tag: v1.6.1
+    tag: v2.0.0
```

Comment:
DB statefulset needs to be modified to handle this image which has new PostgreSQL

Reply:
Handled